cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflow.java

```diff
@@ -126,4 +128,61 @@ protected final void addSpark(String spark) {
     Predicate<WorkflowContext> predicate) {
     return configurer.condition(predicate);
   }
+
+  /**
+   * Adds a local dataset instance to the {@link Workflow}. Local datasets are created at the
+   * start of every {@code Workflow} run and deleted once the run is complete. User can decide to
+   * keep the local datasets even after the run is complete by specifying the appropriate runtime
```

Comment:
What are "appropriate runtime arguments"? Possible to document it here as well (or at least point to somewhere that has it documented, may be in the Javadoc of the Workflow class?)